app/views/maths_and_physics/claims/eligibility_confirmed.html.erb

```diff
@@ -5,6 +5,7 @@
     <h1 class="govuk-heading-xl">You are eligible to claim a payment for teaching maths or physics</h1>
     <p class="govuk-body">Based on your answers you can claim £2,000 for teaching maths or physics.</p>
 
-    <%= link_to "Continue", claim_path(current_policy_routing_name, next_slug), class: "govuk-button", role: "button", data: { module: "govuk-button" } %>
+    <%= button_to "Continue", claim_path(current_policy_routing_name, next_slug), method: :get, class: "govuk-button", data: { module: "govuk-button" } %>
```

Comment:
Can we drop the `data: { module: "govuk-button" }` on this now?

Reply:
Facepalm, yes, yes we can and we will! 👍